src/redux/endpoints/nfts.endpoint.ts

```diff
@@ -19,7 +19,7 @@ const nftsEndpoints = RootApi.injectEndpoints({
         baseURL: getBaseURL(),
         url: `/${ACCOUNTS_ENDPOINT}/${props.address}/${NFTS_ENDPOINT}`,
         method: 'GET',
-        params: { size: MAX_API_SIZE, ...props }
+        params: { excludeMetaESFDT: true, size: MAX_API_SIZE, ...props }
```

Comment:
As far as I know, the parameter should be called `excludeMetaESDT`? Looks like a typo here.

Reply:
Correct

Reply:
Sweet. 🔥